tests/tools/s3/resource_manifest.json

```diff
@@ -51,5 +51,132 @@
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-hello-busybox",
+      "relative_path": "kernel/bzimage",
+      "image_format": "bzimage",
+      "halt_after_boot": false
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-hello-busybox",
+      "relative_path": "kernel/pe",
+      "image_format": "pe",
+      "halt_after_boot": false
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-focal",
+      "relative_path": "kernel/bzimage",
+      "image_format": "bzimage",
+      "halt_after_boot": false,
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-focal",
+      "relative_path": "kernel/elf",
+      "image_format": "elf",
+      "halt_after_boot": false,
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "disk",
+      "resource_name": "ubuntu-focal-rootfs.ext4",
+      "relative_path": "disk/",
+      "distro": "Ubuntu",
+      "compatible_kernels": "v1/kernel/bzimage/bzimage-focal|v1/kernel/elf/vmlinux-focal"
+    }
+  ],
+  "v3": [
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-hello-busybox-halt",
+      "relative_path": "kernel/bzimage",
+      "image_format": "bzimage",
+      "halt_after_boot": true
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-hello-busybox-halt",
+      "relative_path": "kernel/elf",
+      "image_format": "elf",
+      "halt_after_boot": true
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-hello-busybox-halt",
+      "relative_path": "kernel/pe",
+      "image_format": "pe",
+      "halt_after_boot": true
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-hello-busybox",
+      "relative_path": "kernel/elf",
+      "image_format": "elf",
+      "halt_after_boot": false
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-hello-busybox",
+      "relative_path": "kernel/bzimage",
+      "image_format": "bzimage",
+      "halt_after_boot": false
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "pe-hello-busybox",
+      "relative_path": "kernel/pe",
+      "image_format": "pe",
+      "halt_after_boot": false
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "bzimage-focal",
+      "relative_path": "kernel/bzimage",
+      "image_format": "bzimage",
+      "halt_after_boot": false,
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "kernel",
+      "resource_name": "vmlinux-focal",
+      "relative_path": "kernel/elf",
+      "image_format": "elf",
+      "halt_after_boot": false,
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "disk",
+      "resource_name": "ubuntu-focal-rootfs.ext4",
+      "relative_path": "disk/",
+      "distro": "Ubuntu",
+      "compatible_kernels": "v1/kernel/bzimage/bzimage-focal|v1/kernel/elf/vmlinux-focal"
```

Comment:
Can you also add the following intird configuration:
```
    {
      "resource_type": "initrd",
      "resource_name": "aarch64-initramfs.cpio",
      "relative_path": "kernel/pe"
    }
```

Reply:
Added it.

Reply:
Should we add it to v2 as well?

Reply:
I think we shouldn't. I'll remove it from the v2 directory in S3, that was a mistake on my part, sorry.